e2e/README.md

```diff
@@ -17,4 +17,42 @@ You can also run from the molgenis-emx2 project root as follows
 
 ```npx playwright test --config e2e --project=chromium```
 
-The test is part of .circleci/config.yml running that same command
\ No newline at end of file
+The test is part of .circleci/config.yml running that same command
+
+## Catalogue manual tests
+
+The folder `e2e/catalogue` contains the steps taken by cohort data managers to test the '[new design](https://data-catalogue.molgeniscloud.org/catalogue/ssr-catalogue)' catalogue with multiple user roles in mind. Some of these manual steps will overlap the automated steps described above but we aim to minimize overlap. The tests will be performed on the data catalogue [accept](https://data-catalogue-acc.molgeniscloud.org/catalogue/catalogue/#/networks-catalogue) server which the cohort team updates to a specific version. Only when this version is considered stable the update will be rolled out to the data catalogue production server.
```

Comment:
maybe  to use the emx2.dev server and demo data instead of project specific data ? ( also keep in mind the UMCG cohorts use cases )

Reply:
Added the umcg and dev server to the list of potential servers to test